girder_utils/templatetags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter()
+def lookup(value, arg):
+    return value.get(arg, '')
```

Comment:
Why return `''` by default, and not `None`?

Reply:
because template tags should always return strings. See https://github.com/django/django/blob/76c0b32f826469320c59709d31e2f2126dd7c505/django/template/defaultfilters.py.